### ISSUE TITLE
Doc issue: Documentation shows wrong function to call.

### ISSUE THREAD

**Issue by bangerth** [taskflow/taskflow#577]
The examples at https://taskflow.github.io/taskflow/AsyncTasking.html#LaunchAsynchronousTasksFromARuntime , showing how to use `tf::Runtime::async()` then go on to suggest calling `rt.join()`. I think this is a copy-paste issue from above, where the `join()` function was called on sub-flows. But `tf::Runtime` does not have a `join()` member function. I *think* the code snippets should show calling `corun_all()` instead -- which would then also be consistent with the documentation that intersperses the code snippets.

**Comment by tsung-wei-huang** [comment]
Yes, it should be corun_all - thank you for the catch! Feel free to help submit a pull request :)
